--- rtaudio-4.1.2/configure.ac.orig	2016-02-22 22:15:52.000000000 +0100
+++ rtaudio-4.1.2/configure.ac	2016-02-27 20:10:10.924996893 +0100
@@ -149,7 +149,7 @@
     api="$api -D__LINUX_PULSE__"
     req="$req libpulse-simple"
     AC_MSG_RESULT(using PulseAudio)
-    AC_CHECK_LIB(pulse-simple, pa_simple_flush, , AC_MSG_ERROR(PulseAudio support requires the pulse-simple library!))], )
+    AC_CHECK_LIB(pulse-simple, pa_simple_flush, [LIBS="$LIBS $(pkg-config --libs libpulse-simple)"], AC_MSG_ERROR(PulseAudio support requires the pulse-simple library!))], )
 
   # Look for OSS flag
   AC_ARG_WITH(oss, [  --with-oss = choose OSS API support (linux only)], [
